pkg/tfbridge/provider.go

```diff
@@ -517,7 +517,7 @@ func (p *Provider) Configure(ctx context.Context,
 
 	configMap, err := configEnc.UnmarshalProperties(req.GetArgs())
 	if err != nil {
-		return nil, err
+		return nil, ReplaceErrorProperties(err, p.info)
```

Comment:
I don't believe that configEnc.UnmarshalProperties will ever surface missing config errors; is this needed?